checkov/terraform_json/runner.py

```diff
@@ -0,0 +1,230 @@
+from __future__ import annotations
+
+import logging
+import os
+from typing import TYPE_CHECKING, Any
+
+from checkov.common.bridgecrew.check_type import CheckType
+from checkov.common.checks_infra.registry import get_graph_checks_registry
+from checkov.common.graph.graph_builder import CustomAttributes
+from checkov.common.graph.graph_builder.consts import GraphSource
+from checkov.common.output.graph_record import GraphRecord
+from checkov.common.output.record import Record
+from checkov.common.output.report import Report
+from checkov.common.runners.base_runner import CHECKOV_CREATE_GRAPH
+from checkov.common.util.consts import START_LINE, END_LINE
+from checkov.common.util.secrets import omit_secret_value_from_checks
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.checks.resource.registry import resource_registry
+from checkov.terraform.graph_builder.local_graph import TerraformLocalGraph
+from checkov.terraform.runner import Runner as TerraformRunner
+from checkov.terraform_json.utils import get_scannable_file_paths, TF_JSON_POSSIBLE_FILE_ENDINGS, create_definitions
+
+if TYPE_CHECKING:
+    from checkov.common.graph.checks_infra.registry import BaseRegistry
+    from checkov.terraform.graph_manager import TerraformGraphManager
+    from checkov.common.typing import LibraryGraphConnector, _CheckResult
+
+logger = logging.getLogger(__name__)
+
+
+class Runner(TerraformRunner):
```

Comment:
Should probably be called `TerraformJsonRunner` to avoid multiple instances of the name `Runner` in the code